documents/2-RTB/Documentazione_Interna/Norme_di_Progetto.typ

```diff
@@ -772,8 +779,91 @@ Per la realizzazione della documentazione sono stati scelti dal team i seguenti
+// Da rivedere prima di approvazione
+- *`1-Candidatura`*:
+  - *`Verbali_Esterni`*;
+  - *`Verbali_Interni`*;
+  - `Dichiarazione_Degli_Impegni_V1.0.pdf`;
+  - `Lettera_di_presentazione_V1.0.pdf`;
+  - `Valutazione_Capitolati_V1.0.pdf`.
 
+- *`2-RTB`*:
+  - *`Documentazione_Esterna`*:
+    - *`Verbali_Esterni`*;
+    - `Analisi_dei_Requisiti_v1.0.0.pdf`;
+    - `Glossario_v1.0.pdf`;
+    - `Piano_di_Progetto_v1.0.0.pdf`;
+    - `Piano_di_Qualifica_v1.0.0.pdf`.
+    
+  - *`Documentazione_Interna`*:
+    - *`Verbali_Interni`*;
+    - `Norme_di_Progetto_v1.0.0.pdf`.
+
+- *`3-PB`*
+
+==== Struttura della repository BuddyBot
+In questa repository utilizziamo due branch principali:
+
+- *`master`*: contiene l'ultima versione stabile del codice.
+- *`develop`*: raccoglie le ultime modifiche in corso di sviluppo.
+
+Le modifiche apportate in `develop` vengono generalmente integrate in `master` al termine di ogni periodo di lavoro.
+
+La struttura della repository è organizzata come segue, con i nomi delle directory indicati in *grassetto*:
+// Da rivedere prima di approvazione
+- *`PoC`*
+
+==== Sito Vetrina
+La repository #link("https://github.com/SweeTenTeam/SweeTenTeam.github.io","SweeTenTeam.github.io") ospita il codice del sito dedicato al progetto. Il sito ha l'obiettivo di fornire un'interfaccia veloce e intuitiva, nella quale poter visualizzare ergonomicamente e in maniera organizzata i documenti relativi al progetto stesso. Tra le funzionalità del sito, è inclusa anche la visualizzazione diretta del #link("https://sweetenteam.github.io/docs/RTB/Glossario","Glossario"), che fornisce definizioni chiare e precise dei termini rilevanti. Questa sezione aiuta a prevenire fraintendimenti o interpretazioni ambigue e permette di accedere rapidamente alle definizioni cliccando sui termini presenti nei documenti.
+
+=== Sincornizzazione
+La sincronizzazione avviene attraverso #glossary("repository") condivise su #glossary("GitHub"), dove ogni attività è tracciata tramite un'apposita #glossary("issue"). All'inizio di ogni task, il membro del team crea un nuovo #glossary("branch") a partire da *`develop`*. Questo approccio consente di avere ambienti di lavoro indipendenti, permettendo di svolgere più attività in parallelo e ottimizzando la collaborazione all'interno del team.
 
+==== Branch
+La struttura dei branch descritta in precedenza si basa sulla metodologia single-purpose, che prevede la creazione di un #glossary("branch") per ciascuna attività specifica. Questo approccio consente di suddividere il lavoro in compiti distinti, garantendo un flusso di lavoro stabile e organizzato. Una volta completata un'attività, il #glossary("branch") corrispondente viene unito al "branch principale" (*`develop`*) e successivamente eliminato. È importante chiarire che *`master`* è il "branch di default", mentre per "branch principale" si intende il #glossary("branch") in cui le modifiche devono essere integrate.
```

Comment:
Puoi specificare, quando parli del master, che è il branch di default utilizzato solo per la visione dei PDF compilati, come dice sempre a lezione il prof. Mentre sul develop, come già hai menzionato in altre parti, ci sta tutto il sorgente.

Reply:
Qui lascerei invariato perchè si parla dei branch in maniera generale, non nello specifico di quelli della repo docs ma anche quelli della repo BuddyBot

Reply:
roger